pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy

```diff
@@ -247,9 +250,26 @@ public class ModelInterpreter implements Serializable {
         }
     }
 
+    private Map<String,CredentialWrapper> processCredentials(@Nonnull List<List<String>> varsAndIds) {
```

Comment:
`List<List<String>>` is a very convoluted data structure.
`Map.Entry` is serializable iirc.